models/comments.models.js

```diff
@@ -0,0 +1,14 @@
+const db = require('../db/connection.js')
+
+async function removeCommentById(comment_id) {
+    const validCommentIDs = await db.query(`SELECT comment_id FROM comments;`).then((comments) => {
+        return comments.rows.map((comment) => comment.comment_id);
+    });
+    if (!isNaN(+comment_id) &&!validCommentIDs.includes(Number(comment_id))) return Promise.reject({ status: 404, message: 'Comment id not found' })
+    return db.query(`DELETE FROM comments WHERE comment_id = $1;`, [comment_id]).then(() => {
+        return;
+    });
```

Comment:
Yes i did think two were uneccesary, but I thought I might have to because i should get an empty array back for a valid one (because it's been deleted), but i might be wrong.